Userland/Libraries/LibHTTP/HttpJob.cpp

```diff
@@ -24,13 +24,13 @@ void HttpJob::start(NonnullRefPtr<Core::Socket> socket)
     };
     m_socket->set_idle(false);
     if (m_socket->is_connected()) {
-        dbgln("Reusing previous connection for {}", url());
+        dbgln_if(CHTTPJOB_DEBUG, "Reusing previous connection for {}", url());
```

Comment:
I wonder what a CHTTP is...

Reply:
Me too! Maybe a remnant from the olden times in serenity, when classes started with `C`?

Reply:
Remnant of when it was `Core::HttpJob` (before just about the same time LibTLS came to be I think)

Reply:
dropping the `C` in #12084, thanks.